src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_adls_gen2_scenarios.py

```diff
@@ -48,6 +48,177 @@ def test_adls_access_scenarios(self, resource_group, storage_account):
+        filesystem = self.create_file_system(account_info)
+
+        dir0 = self.create_random_name(prefix="dir0", length=12)
+        subdir0 = '/'.join([dir0, 'dir0'])
+        subdir1 = '/'.join([dir0, 'dir1'])
+        subdir2 = '/'.join([dir0, 'dir2'])
+
+        file1 = '/'.join([subdir1, self.create_random_name(prefix="file1", length=12)])
+        file2 = '/'.join([subdir1, self.create_random_name(prefix="file2", length=12)])
+        file3 = '/'.join([subdir1, self.create_random_name(prefix="file3", length=12)])
+        file4 = '/'.join([subdir2, self.create_random_name(prefix="file4", length=12)])
+        file5 = '/'.join([subdir2, self.create_random_name(prefix="file5", length=12)])
+        file6 = '/'.join([subdir2, self.create_random_name(prefix="file6", length=12)])
+
+        local_file = self.create_temp_file(16)
+
+        # Prepare files
+        self.oauth_cmd('storage fs directory create -n "{}" -f {} --account-name {} '.format(
+            dir0, filesystem, storage_account))
+        self.oauth_cmd('storage fs directory create -n "{}" -f {} --account-name {} '.format(
+            subdir0, filesystem, storage_account))
+        self.oauth_cmd('storage fs directory create -n "{}" -f {} --account-name {} '.format(
+            subdir1, filesystem, storage_account))
+        self.oauth_cmd('storage fs directory create -n "{}" -f {} --account-name {} '.format(
+            subdir2, filesystem, storage_account))
+
+        self.storage_cmd('storage fs file upload -p "{}" -f {} -s "{}" ', account_info, file1, filesystem, local_file)
+        self.oauth_cmd('storage fs file upload -p "{}" -f {} -s "{}" --account-name {} '.format(
+            file2, filesystem, local_file, storage_account))
+        self.oauth_cmd('storage fs file upload -p "{}" -f {} -s "{}" --account-name {} '.format(
+            file3, filesystem, local_file, storage_account))
+        self.oauth_cmd('storage fs file upload -p "{}" -f {} -s "{}" --account-name {} '.format(
+            file4, filesystem, local_file, storage_account))
+        self.storage_cmd('storage fs file upload -p "{}" -f {} -s "{}" ', account_info, file5, filesystem, local_file,
+                         storage_account)
+        self.oauth_cmd('storage fs file upload -p "{}" -f {} -s "{}" --account-name {} '.format(
+            file6, filesystem, local_file, storage_account))
+
+        # check permissions
+        self.storage_cmd('storage fs access set -p "/" -f {} --permissions 111 ', account_info, filesystem)
+        items = self.storage_cmd('storage fs file list -f {}  --exclude-dir --query [].name ',
+                                 account_info, filesystem).get_output_in_json()
+        for item in items:
+            self.storage_cmd('storage fs access set -p "{}" -f {} --permissions rwxr-x--- ', account_info, item,
+                             filesystem)
+
+        acl1 = "default:user:21cd756e-e290-4a26-9547-93e8cc1a8923:rwx"
```

Comment:
> 21cd756e-e290-4a26-9547-93e8cc1a8923

May I ask if this ID can be generated dynamically instead of fixed?

Reply:
yes